generators/client/templates/src/test/javascript/spec/app/account/reset/finish/_reset.finish.controller.spec.js

```diff
@@ -54,7 +54,7 @@ describe('Controller Tests', function() {
 
             // then
             expect(MockTimeout).toHaveBeenCalledWith(jasmine.any(Function));
-            expect(MockAngular.element).toHaveBeenCalledWith('[ng-model="vm.resetAccount.password"]');
```

Comment:
did you try this `expect(MockAngular.element).toHaveBeenCalledWith('#password');` ?